nixos/modules/services/mail/postfix.nix

```diff
@@ -626,6 +715,10 @@ in
           The main.cf configuration file as key value set.
 
           Null values will not be rendered.
+
+          ::: {.tip}
+          Check `postconf -d` for the default values of all settings.
```

Comment:
Is this the reason you don't want to hardcode upstream's default value as our default, even where it's not generated from other parameters?

Reply:
I made the main.cf renderer drop null values in 951a020ed4ab8a684c2a13c8c178ba1cb7785b6d so there was the option to just use the default.

Reply:
Yeah, makes sense.